kratos/geometries/brep_curve_on_surface.h

```diff
@@ -172,6 +207,21 @@ class BrepCurveOnSurface
         return typename BaseType::Pointer( new BrepCurveOnSurface( ThisPoints ) );
     }
 
+    ///@}
+    ///@name Set/ Get functions
+    ///@{
+
+    bool GetCurveDirection()
+    {
+        return mCurveDirection;
```

Comment:
maybe here sth like you use in #6215 would make sense here too

Reply:
It is something coming from the input. And allows only, same direction or controverse direction.

Reply:
For me at least the name of the method is confusing and the return value deosn't help. What is the meaning of ture or false?

Reply:
Didn't you find the comment declarative enough? ;)

I changed it to more convenient name mSameCurveDirection and added some comments.

Reply:
@philbucher do you agree with the new name? Is it more understandable?